main.go

```diff
@@ -67,6 +68,16 @@ func setupGlobalFlags() {
 	flag.StringVar(&targetDb, "target-db", conversion.TARGET_SPANNER, "target-db: Specifies the target DB. Defaults to spanner")
 }
 
+func didSetVerboseAlready() bool{
```

Comment:
can you think of a better name for this function - something around setVerboseTwice or similar ?